src/ui/tooltip.gd

```diff
@@ -0,0 +1,29 @@
+extends Control
+
+
+@onready var info_text: Label = %InfoText
+@onready var damage_amt: Label = %DamageAmt
+@onready var health_amt: Label = %HealthAmt
+@onready var move_amt: Label = %MoveAmt
+@onready var damage_icon: TextureRect = %DamageIcon
+@onready var health_icon: TextureRect = %HealthIcon
+@onready var move_icon: TextureRect = %MoveIcon
+
+
+func initialize(unit : Unit) -> void:
```

Comment:
Super pedantic:
```gdscript
func initialize(unit: Unit) -> void:
```
(no space in between `unit` and `:`, I am very sorry)

Reply:
![image](https://github.com/kdreese/gmtk-2023/assets/34196410/201ae612-db5b-48ee-803b-783419ce38b5)